src/SOS/Strike/strike.cpp

```diff
@@ -15362,6 +15362,74 @@ _EFN_GetManagedObjectFieldInfo(
     return S_OK;
 }
 
+DECLARE_API(VerifyGMT)
```

Comment:
It feels a little unfortunate that we are adding code to SOS whose only purpose is for testing. I think its OK on a small scale, but if we find ourselves repeating this pattern too often we should probably brainstorm a way to get this code out of the product bits and into the tests. None of the solutions I have thought of thus far are quick/cheap so I'm fine with taking a wait and see approach.

Reply:
It is like VerifyStackTrace which tests the EFN_GetStackTrace/EFN_GetManagedExcepStack code, but I'm fine with removing it.

Reply:
I think its better to keep it here if the alternative is not testing it.